tests/app/questionnaire/test_placeholder_transforms.py

```diff
@@ -38,6 +38,38 @@ def test_format_number(number, expected, transformer):
     assert transformer().format_number(number) == expected
 
 
+@pytest.mark.parametrize(
+    "value, expected",
+    (
+        (123, "123%"),
+        ("123.4", "123.4%"),
+        ("123.40", "123.40%"),
+        ("1000", "1000%"),
+        (0, "0%"),
+        (0.00, "0.0%"),
+        ("", "%"),
```

Comment:
When would we get an empty answer?. We should remove this as we don't store empty value.

Reply:
Removed the empty answer, thanks!